INSTALL.md

```diff
@@ -96,6 +96,13 @@ Installing Mirage:
 2. [Unmask](https://wiki.gentoo.org/wiki/Knowledge_Base:Unmasking_a_package) `net-im/mirage`
 3. Run `emerge net-im/mirage`
 
+#### Alpine Linux
+
+If you are on the Edge channel of Alpine Linux, Mirage can be installed
```

Comment:
We can add instructions or a link to setup the edge channel, maybe https://wiki.alpinelinux.org/wiki/Edge?

Reply:
Done. How does this sound?

Reply:
I would change "you can do so here: https://wiki.alpinelinux.org/wiki/Edge" to something like "you can do so on the [Alpine wiki](https://wiki.alpinelinux.org/wiki/Edge)", otherwise good